### DISCUSSION TITLE
Homepage download link

### DISCUSSION THREAD

**Discussion by CGenie** [zellij-org/zellij#3465] [General]
Hello,

It's kind of annoying that https://zellij.dev has a download link of this form:
```html
<button type="button" class="download-hover" onclick="location.href='https://github.com/zellij-org/zellij/releases/latest/download/zellij-x86_64-unknown-linux-musl.tar.gz'">
                linux download
              </button>
```
For a CLI app I expect just a simple `a href=...` so that I can easily copy-paste it into terminal...

**Comment by Lockszmith-GH** [comment]
What's wrong with this?
![image](https://github.com/zellij-org/zellij/assets/905716/84499497-fe95-4014-bf33-fe9f18e554d5)

Seems simple enough, just need to type a short cli

**Reply by CGenie** [reply]
I don't know, many people object the `bash`/`curl` combo because of safety. And, again, I think most people want to install it on remote servers instead of downloading locally.

**Reply by Lockszmith-GH** [reply]
For those I actually have my own script where I just supply the github project, and it grabs the files from the lastest release tag for me.
We're command line users and scripters here :) trust your scripts.

But I can get a recommendation for simplifying the rendering of the front page.
We'll leave it to the maintainer of the project to decide.

**Comment by Rasmus-Bertell** [comment]
Generally speaking having links in button elements is not very accessible, for example that link won't work if someone has JS disabled while browsing.
